src/sentry/event_manager.py

```diff
@@ -936,6 +936,10 @@ def _save_aggregate2(event, flat_hashes, hierarchical_hashes, release, **kwargs)
     existing_group_id = _find_group_id(all_hashes)
 
     if existing_group_id is None:
+
+        if project.id in (options.get("store.load-shed-group-creation-projects") or ()):
+            raise HashDiscarded("Load shedding group creation")
```

Comment:
@untitaker Is it applicable here, or had not been added intentionally?

Reply:
this is fine